api/v3/openapi.yaml

```diff
@@ -322,6 +322,50 @@ paths:
         '200':
           $ref: '#/components/responses/ListConsumersResponse'
 
+  /clusters/{cluster_id}/consumer-groups/{consumer_group_id}/lag:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+      - $ref: '#/components/parameters/ConsumerGroupId'
+
+    get:
+      summary: 'Get Consumer Group Lag.'
+      description: 'Returns the max and total lag of the consumers belonging to the specified consumer group.'
+      tags:
+        - Consumer Group
+      responses:
+        '200':
+          $ref: '#/components/responses/GetConsumerGroupLagResponse'
+
+  /clusters/{cluster_id}/consumer-groups/{consumer_group_id}/lags/{topic_name}/partitions/{partition_id}:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+      - $ref: '#/components/parameters/ConsumerGroupId'
+      - $ref: '#/components/parameters/TopicName'
+      - $ref: '#/components/parameters/PartitionId'
+
+    get:
+      summary: 'Get Consumer Lag'
+      description: 'Returns the consumer lag on a partition with the given `partition_id`.'
+      tags:
+        - Partition
+      responses:
+        '200':
+          $ref: '#/components/responses/GetConsumerLagResponse'
+
+  /clusters/{cluster_id}/consumer-groups/{consumer_group_id}/lags:
```

Comment:
You should move this endpoint before `/clusters/{cluster_id}/consumer-groups/{consumer_group_id}/lags/{topic_name}/partitions/{partition_id}` to keep them lexicographically sorted.